software/forgejo.yml

```
@@ -0,0 +1,15 @@
+name: Forgejo
+website_url: https://forgejo.org
+source_code_url: https://codeberg.org/forgejo/forgejo/
+description: A lightweight software forge maintained by an inclusive community under the umbrella of the non-profit Codeberg e.V. and focused on scaling, federation, and privacy.
+licenses:
+  - MIT
+platforms:
+  - Go
```

> https://forgejo.org/docs/latest/admin/installation/

As Docker is first, its listed first.

```suggestion
platforms:
 - Docker
   - Go
```

---

Thanks for the suggestion @KieranRobson, I've made that change.